agent/main.go

```diff
@@ -126,5 +119,8 @@ func main() {
 		log.G(ctx).WithError(err).Warn("shim error")
 	}
 
-	log.G(ctx).Info("done")
+	log.G(ctx).Info("shutting down agent")
+	if _, err := runcTaskService.Shutdown(ctx, &shimapi.ShutdownRequest{ID: id, Now: true}); err != nil {
+		log.G(ctx).WithError(err).Error("runc shutdown error")
```

Comment:
maybe modify this error to be more clear that we errored calling something else, not runc.

Reply:
This error message is not hittable atm and I don't have a good alternative to rephrase this is a meaningful way so LGTM.